src/components/sbb-button/sbb-button.tsx

```diff
@@ -33,12 +29,23 @@ export class SbbButton {
   @Prop() public eventId?: string;
 
   /** Define if icon should be shown or not */
-  @Prop() public icon? = false;
+  @Prop() public icon? = true;
+
+  /**
+   * The icon name we want to use,
+   * choose from the small icon variants from
+   * the ui-icons category from here
+   * https://lyne.sbb.ch/tokens/icons/ (optional).
+   * Inline variant doesn't support icons.
+   */
+  @Prop() public iconName?: string;
 
   /** If you use an icon without a label, you must provide an iconDescription */
   @Prop() public iconDescription?: string;
```

Comment:
I wonder if we still need the `iconDescription` prop for screen-readers only, since we can now handle the icon accessibility through the `sbb-icon` component itself in case we intend to use it, and I think we should (doc: [https://github.com/lyne-design-system/lyne-components/tree/master/src/components/sbb-icon#accessibility](url)).

What do you think (@kyubisation @jeripeierSBB)? Will it be expected to use the `sbb-icon` component here as well, changing the current behavior?

Reply:
Yes, I would agree. A consumer can slot a `<sbb-icon>` with aria information if it is required.

Reply:
I removed the icon Description, if a description is required, for example in case of an button with icon only, it should be set via accessibility-label (I added a note to the readme) prop on the button or direct on the icon if its slotted.